oc-chef-pedant/Gemfile

```diff
@@ -6,15 +6,16 @@ gem "pry"
 gem "pry-byebug"
 gem "pry-stack_explorer"
 gem "rake"
-gem "rest-client", git: "https://github.com/chef/rest-client", branch: "jfm/ucrt_update1"
+gem "rest-client", git: "https://github.com/chef/rest-client.git", branch: "jfm/ucrt_update1"
 
 # We require chef internally to get the version in
 # lib/pedant/request.rb. It's really strange code and it should be
 # revisited.
-gem "chef", "18.8.46"
+# gem "chef", "18.8.46" - commmenting this out to get past loading that enormous gem to only get a version number.
```

Comment:
Should be removed instead of keeping commented code to avoid confusion in future. We can always refer this back from commit history

Reply:
removed